### PR TITLE
chore: update biome configuration and scripts

### PR DESCRIPTION
**Pull request by ematipico**
## Summary
The `biome/**` was causing weird issue on my machine. After I deleted it, the scripts started to work again

Closes #533

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 .astro/
 dist
-node_modules
```
Comment (Sec-ant) [inline .gitignore:3]: We shouldn't remove `node_modules`.

Comment (ematipico) [reply .gitignore:3]: Why? Biome ignores it already

Comment (Sec-ant) [reply .gitignore:3]: But this is for `git` to ignore it, isn't it?

Comment (ematipico) [reply .gitignore:3]: Lol true 😅

**Review by Sec-ant — APPROVED** [review]
This closes #533

**Pull request merged**